packages/vite/src/node/server/middlewares/htmlFallback.ts

```diff
@@ -51,7 +37,7 @@ export function htmlFallbackMiddleware(
       const filePath = path.join(root, pathname)
       if (fs.existsSync(filePath)) {
         debug?.(`Rewriting ${req.method} ${req.url} to ${url}`)
-        req.url = prepend + url
+        req.url = url
```

Comment:
```suggestion
```
I guess this assignment does nothing.

Reply:
Technically it removes search queries as it's been called with `cleanUrl()`. I'm thinking we could keep it for consistency with the other conditions 🤔

Reply:
That makes sense. 👍